spec/requests/mc_responses_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe "McResponses", type: :request do
+
+  describe "positive tests" do 
+
+    let!(:survey) {Survey.create(title: "test", id: 1, isLive: false, wentLiveAt: nil)}
+    let!(:mc_question_test) {McQuestion.create(question: "test question", survey_id: survey.id, id: 1), mc_options: {options: ["op1", "op2"]}}
```

Comment:
You can't create mc_options this way, they are a separate database entry so they need their own let.

Reply:
note, the let! statement pretty much just creates a spoof object and database entry. So when you are creating a spoof object with let the creation is the same as when you create a normal object. Sorry I should have noticed this error earlier.